lapce-data/src/document.rs

```diff
@@ -470,6 +471,11 @@ impl Document {
         };
         let mut selection_find = Find::new(0);
         selection_find.case_matching = CaseMatching::Exact;
+        let config = LapceConfig::load(&LapceWorkspace::default(), &[]);
+        let parser = BracketParser::new(
+            "".to_string(),
+            config.editor.bracket_pair_colorization,
+        );
```

Comment:
This is wrong. It should load config from calling function

Reply:
You mean pass from the new function of Document?